docs/friends.md

```diff
@@ -0,0 +1,100 @@
+# Friends
+
+Friends are a great way to build a social community. Users can add other users to their list of friends, see who is online or when they were last online, chat together in realtime, and interact together in gameplay or collaboration.
+
+Each user builds up a list of friends by who they know already from the social networks, friend requests they send, requests they receive, and who the server recommends they should know. This information is stored in a social graph within the system as a powerful way to interact with other users. Much like how Twitter or Facebook work.
+
+!!! success "Fun fact"
+    Nakama is a common Japanese word that directly translates to friend or comrade. Some believe the word means "people who are considered closer than family", though that is not a part of the official definition. We feel it expresses the kind of social communities we want developers to build into their games and apps!
+
+Any social community must be maintained carefully to prevent spam or abuse. To help with this problem it's also possible for a user to block users they no longer want to communicate with and for the server to ban a user via server-side code to completely disable an account.
+
+## Add friends
+
+A user can add one or more friends by that user's ID or handle. The user added will not be marked as a friend in the list until they've confirmed the friend request. The user who receives the request can confirm it by adding the user back.
+
+!!! hint
+    A user who registers or links their account with Facebook or another social network will have friends from that network be added automatically into their friend list.
+
+When a friend request is sent or the user is added an in-app notification will be sent. See the [in-app notification](in-app-notifications.md#receive-notifications) section for more info.
+
+```csharp fct_label="Unity"
+byte[] userId = ...; // some user ID
+var message = NFriendAddMessage.Default(userId);
+client.Send(message, (bool done) => {
+  Debug.Log("Friend added or request sent.");
+}, (INError err) => {
+  Debug.LogErrorFormat("Error: code '{0}' with '{1}'.", err.Code, err.Message);
+});
+```
+
+When both users have added eachother as friends they are able to initiate realtime chat in a 1-on-1 channel. See the [realtime chat](realtime-chat.md) section for more info.
```

Comment:
Is this actually true? We don't prevent people from messaging each other when they are not yet friends.